obspy/geodetics/base.py

```diff
@@ -315,37 +315,42 @@ def locations2degrees(lat1, long1, lat2, long2):
     Earth. For more accurate values use the geodesic distance calculations of
     geopy (https://github.com/geopy/geopy).
 
-    :type lat1: float
-    :param lat1: Latitude of point 1 in degrees
-    :type long1: float
-    :param long1: Longitude of point 1 in degrees
-    :type lat2: float
-    :param lat2: Latitude of point 2 in degrees
-    :type long2: float
-    :param long2: Longitude of point 2 in degrees
-    :rtype: float
-    :return: Distance in degrees as a floating point number.
+    :type lat1: ``float or :class:`numpy.ndarray```
```

Comment:
Minor nitpick: Can you please remove the double-backticks around the whole type? This page describes the syntax of the sphinx info field lists: http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists